src/richie/apps/courses/templates/courses/cms/category_detail.html

```diff
@@ -152,7 +152,7 @@ <h2 class="category-detail__title">{% trans "Related organizations" %}</h2>
                                 {% include "courses/cms/fragment_organization_glimpse.html" %}
                             {% endfor %}
                             {% if paginator.num_pages > 1 %}
-                                {% paginate using "richie/pagination.html" %}
+                                {% include "richie/pagination.html" with label="Related organizations pagination" %}
```

Comment:
If you do like this, it will not get translated. This should make the job:
```suggestion
                                {% trans "Custom pagination" as pagination_label %}
                                {% include "richie/pagination.html" with label=pagination_label %}
```

Reply:
Oh yeah sorry. Would it be okay to use the `_()` syntax to more concise?

```diff
- {% trans "Custom pagination" as pagination_label %}
- {% include "richie/pagination.html" with label=pagination_label %}
+ {% include "richie/pagination.html" with label=_("Custom pagination") %}
```

Reply:
If it works OK! Never seen it though

Reply:
Tested it locally, it does work. Saw this [in the django docs](https://docs.djangoproject.com/fr/4.0/topics/i18n/translation/#string-literals-passed-to-tags-and-filters) :)

Reply:
Cool 🤩